mlir/lib/Dialect/MIOpen/Tuning/GridwiseGemmParams.cpp

```diff
@@ -176,7 +176,16 @@ LogicalResult PopulateParams::paramsFromCtx(
     LLVM_DEBUG(llvm::dbgs() << "FATAL ERROR! COULD NOT FIND VALID TUNING"
                             << " PARAMETERS!\n");
 
-    if (ctx.opType != miopen::ConvOpType::Conv2DBwdDataOpType) {
+    bool isPaddingKernelSupport = false;
```

Comment:
I feel the logic is duplicated between `PopulateParamsXDL` and `PopulateParams`. Please try extract them out as a lambda or a utility function.